torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1455,6 +1455,9 @@ Error PyTorchModelLoader::loadQuantizedLinear(const torch::jit::Node *ptNode) {
   const c10::optional<at::Tensor> ptBiasTensor =
       unpackedParams[1].toOptional<at::Tensor>();
 
+  bool isGroupwiseQuantized = ptWeightTensor.is_quantized() &&
```

Comment:
Why not isChannelwiseQuantized()?

Reply:
Yeah... Actually here it should be isRowwiseQuantized